cmake/linux/package_linux.sh.in

```diff
@@ -9,8 +9,10 @@
 set -e
 
 USERBIN="$HOME/bin"
-LINUXDEPLOYQT="$USERBIN/linuxdeployqt"
-APPIMAGETOOL="$USERBIN/appimagetool"
+LINUXDEPLOYQTDIR="$HOME/bin/linuxdeployqt"
+LINUXDEPLOYQTBIN="$LINUXDEPLOYQTDIR/usr/bin"
+LINUXDEPLOYQT="$LINUXDEPLOYQTBIN/linuxdeployqt"
```

Comment:
`$HOME/bin/foo` is a search path for some of these portable tools and they were chosen that way for a reason.  If fuse isn't supported, `$HOME/bin/foo/bin/foo` doesn't make sense.

Perhaps we just keep a [`squashfs-root`](https://github.com/AppImage/AppImageKit/wiki/FUSE#docker) folder to be consistent with upstream documentation.  Also, according to the documentation, we should always execute the `AppRun` executable to setup the environment properly.  This may not be an issue now, but we don't want to use the app other than how it was intended, and the `AppRun` can be a [wrapper around the launcher](https://github.com/LMMS/lmms/pull/3688/files#diff-5b07d65c1f1bdd29f324c363c12c9c9bR97).

But the more I think about this the more I wonder if we should have the non-fuse logic as some sort of detect-and-fallback so that people that do this locally can continue to have a copy of the binaries in the default `$HOME/bin` location?

Reply:
Another idea (assuming we detect-and-wrap the fuse logic) is that we can also symlink to `$HOME/bin/foo` and point it to the `AppRun`, thus simulating what fuse would have done.  Circle-CI may not be the only desktop with FUSEturned off.  I remember @jasp00 not liking the reliance on FUSE.

Reply:
Here's some inspiration around a wrapper, authored by one of the `appimagetool` maintainers. https://github.com/TheAssassin/redeclipse-appimage/blob/master/build-appimages.sh#L94-L96